baekjoon/큐.js

```diff
@@ -0,0 +1,47 @@
+let fs = require('fs');
+let input = fs.readFileSync('/dev/stdin').toString().split('\n');
+
+class Queue {
+    constructor() {
+        this.arr = [];
+    }
+    enqueue(data) {
+        this.arr.push(data);
+    }
+    dequeue() {
+        return this.arr.length ? this.arr.shift() : -1;
+    }
+    size() {
+        return this.arr.length;
+    }
+    empty() {
+        return this.arr.length ? 0 : 1;
+    }
+    front() {
+        return this.arr.length ? this.arr[0] : -1;
+    }
+    back() {
+        return this.arr.length ? this.arr[this.arr.length - 1] : -1;
+    }
+}
+
+const queue = new Queue();
+
+function solution(input) {
+    const num = Number(input[0]);
+    let answer = 0;
```

Comment:
```suggestion
    let answer = "";
```
이 부분에 0을 넣게 되면 처음 answer에 값이 합해질 때 0 또한 나오게 돼서 오답이 되는 것 같아요 이 부분을 빈문자로 바꿔서 더해주면 정답이 돼서 아쉽네요~
그리고 런타임 오류난 것 공개로 되어 있길래 한 번 봤는데 answer부분을 ""하셨으나  아래 반복문 부분을 0부터 시작해서 숫자인 부분은 .split(" ")으로 나눌 수 없기에 오류가 발생한 것 같아요 
그 외 다른 오류 부분들은 지금 답에서 다 고치셨기에 언급하지 않겠습니다.